.github/workflows/build.yml

```diff
@@ -29,3 +28,28 @@ jobs:
         uses: codecov/codecov-action@v1.0.14
         with:
           file: ./coverage.txt
+      - name: Get image repository
+        run: echo IMAGE_REPOSITORY=$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]') >> $GITHUB_ENV
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
```

Comment:
Are these steps really necessary?

Reply:
QEMU is required. All platforms is not required and you may specify list of platforms, but this step is lasts only 5 seconds.

Reply:
You may also get rid of `IMAGE_REPOSITORY` variable and replace it with `twinproduction/gatus`, but it useful in case someone will create the new fork of your repo.